app/helpers/back-action.js

```diff
@@ -23,7 +23,7 @@ export default Ember.Helper.extend({
 
       const noHistory = window.history.length === 1;
       const deepLinked =
-        document.referrer && document.referrer.indexOf(config.APP.NAME) < 0;
+        document.referrer && document.referrer.indexOf(config.APP.ORIGIN) < 0;
```

Comment:
What does `document.referrer` returns ?

Reply:
It's just a string with the url of the page that redirected to your app.

So if on "jira.com" I click on a `stock.goodcity.hk` link, then referrer will be Jira

Reply:
so if the referrer is Stock, that means I navigated from my app -> to my app

Reply:
Thanks 👍  for clarifying.